### PR TITLE
Doc: Add example of merging a Series with a Dataframe

### LINKED ISSUE #12550: Equivalent of Series.map for DataFrame
I think there should be a way to take a DataFrame and use the values in its columns as indices into a MultiIndexed Series.  Here is an example:

```
>>> d = pandas.DataFrame({"Let": ["A", "B", "C"], "Num": [1, 2, 3]})
>>> d
  Let  Num
0   A    1
1   B    2
2   C    3
>>> ser = pandas.Series(
...     ['a', 'b', 'c', 'd', 'e', 'f'],
...     index=pandas.MultiIndex.from_arrays([["A", "B", "C"]*2, [1, 2, 3, 4, 5, 6]])
... )
>>> ser
A  1    a
B  2    b
C  3    c
A  4    d
B  5    e
C  6    f
dtype: object
```

With this data, you should be able to do `d.map(ser)` (or whatever method name instead of `map`) and get the same result as this:

```
>>> ser.ix[d.apply(tuple, axis=1)]
A  1    a
B  2    b
C  3    c
dtype: object
```

You currently cannot do this without converting the rows to tuples (`ser[d]` gives `ValueError: Cannot index with multidimensional key`).  Converting to tuple is an awkward way to do a very natural task, which is using a tabular array of data to look up values in a tabular index (i.e., a MultiIndex).

I'm creating an issue to resurrect this request from much earlier discussion [here](https://groups.google.com/forum/#!searchin/pydata/dataframe$20map/pydata/sibkQ5Ea-Hc/RKMwiNsH7TMJ) and [here](http://stackoverflow.com/questions/22293683/equivalent-of-series-map-for-dataframe).

### PR DESCRIPTION
**Pull request by pdpark**
- [X] closes #12550

### CODE REVIEW

File: doc/source/merging.rst [hunk 1]
```diff
@@ -714,6 +714,31 @@ either the left or right tables, the values in the joined table will be
           labels=['left', 'right'], vertical=False);
    plt.close('all');
 
+To join a Series and a DataFrame, the Series has to be transformed into a DataFrame first:
```
Comment (jreback) [inline doc/source/merging.rst]: use double backticks around Series & DataFrame

**Review by jreback — CHANGES_REQUESTED** [review]
this isn't referenced to #19140 which is a PR. what is the associated issue?

**Comment by jreback** [comment]
you need to rebase on master and push again, don't see any changes

**Comment by pdpark** [comment]
This PR can be closed - resubmitted as PR #19235.

**Comment by jreback** [comment]
in he future pls just push to thme same PR with changes

**Comment by pdpark** [comment]
I see. Will do.

**Pull request closed without merging**